src/components/TileCatalogNew/TileCatalogNew.jsx

```diff
@@ -30,7 +30,12 @@ const propTypes = {
   /** Call back function of sort */
   onSort: PropTypes.func,
   /** Options in sort */
-  sortOptions: PropTypes.arrayOf(PropTypes.node),
+  sortOptions: PropTypes.arrayOf(
+    PropTypes.shape({
+      id: PropTypes.string,
+      text: PropTypes.string,
+    })
+  ),
```

Comment:
Making a note here - this would normally be a breaking change (major), but TileCatalogNew is still in experimental and so isn't bound by semver 👍